Scripts/AppRemoval/Invoke-ForceRemoveEdge.ps1

```
@@ -44,22 +54,81 @@ function Invoke-ForceRemoveEdge {
+    }
+    catch {
+        Write-Warning "Failed to forcefully uninstall Microsoft Edge: $($_.Exception.Message)"
+        return $false
+    }
+    finally {
+        if ($uninstallRegKey) { $uninstallRegKey.Dispose() }
+        if ($hklm) { $hklm.Dispose() }
+    }
+}
+
+function Remove-EdgeAutostartValue {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Remove-EdgeAutostartValue' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)